tox.ini

```diff
@@ -1,6 +1,13 @@
 [tox]
-envlist = py27, py33, py34, py35, py36
+envlist = {py27,py33,py34,py35,py36}-{pytest,behave}
```

Comment:
This creates ten different tox environments. I think the `--{env, env}` is typically used for testing against different versions of packages. In our case, pytest and behave can be run in the same tox environment. This would speed up testing.

Why don't we add pytest and behave (`cd test && behave`) as the commands to run in every testenv? Then, for `deps`, just do `-rrequirements-dev.txt`